Workflow failure: master

## Workflow Failure

[master](https://github.com/pulumi/pulumi-okta/blob/master/.github/workflows/master.yml) has failed. See the list of failures below:

- [2025-02-26T06:24:02.000Z](https://github.com/pulumi/pulumi-okta/actions/runs/13537704133)

```
=== CONT  TestAccUserTs
    program.go:1961: sample: /home/runner/work/pulumi-okta/pulumi-okta/examples/user
    program.go:1966: pulumi: /opt/hostedtoolcache/pulumi/3.153.0-alpha.xf5bd330/x64/pulumi
    program.go:1094: acquired yarn install lock
    command.go:46: **** Invoke '/usr/local/bin/yarn install' in '/tmp/p-it-fv-az1444--user-c16efb32-540467627'
    command.go:118: Wrote output to /tmp/p-it-fv-az1444--user-c16efb32-540467627/command-output/yarn-install.20250226-062236.86e33.log
    program.go:1120: released yarn install lock
    program.go:1094: acquired yarn install lock
    command.go:46: **** Invoke '/usr/local/bin/yarn link @pulumi/okta' in '/tmp/p-it-fv-az1444--user-c16efb32-540467627'
    command.go:118: Wrote output to /tmp/p-it-fv-az1444--user-c16efb32-540467627/command-output/yarn-link.20250226-062236.a894c.log
    program.go:1120: released yarn install lock
    program.go:2118: projdir: /tmp/p-it-fv-az1444--user-c16efb32-540467627
    program.go:1348: Initializing project (dir /tmp/p-it-fv-az1444--user-c16efb32-540467627; stack p-it-fv-az1444--user-c16efb32)
    command.go:46: **** Invoke '/opt/hostedtoolcache/pulumi/3.153.0-alpha.xf5bd330/x64/pulumi login --cloud-url file:///tmp/TestAccUserTs3957432443/001' in '/tmp/p-it-fv-az1444--user-c16efb32-540467627'
    command.go:118: Wrote output to /tmp/p-it-fv-az1444--user-c16efb32-540467627/command-output/pulumi-login.20250226-062236.69405.log
    command.go:46: **** Invoke '/opt/hostedtoolcache/pulumi/3.153.0-alpha.xf5bd330/x64/pulumi stack init p-it-fv-az1444--user-c16efb32' in '/tmp/p-it-fv-az1444--user-c16efb32-540467627'
    command.go:118: Wrote output to /tmp/p-it-fv-az1444--user-c16efb32-540467627/command-output/pulumi-stack-init.20250226-062237.05495.log
    program.go:1515: Performing primary preview and update
    command.go:46: **** Invoke '/opt/hostedtoolcache/pulumi/3.153.0-alpha.xf5bd330/x64/pulumi preview --non-interactive --diff' in '/tmp/p-it-fv-az1444--user-c16efb32-540467627'
    command.go:118: Wrote output to /tmp/p-it-fv-az1444--user-c16efb32-540467627/command-output/pulumi-preview-initial.20250226-062241.48dbd.log
    command.go:46: **** Invoke '/opt/hostedtoolcache/pulumi/3.153.0-alpha.xf5bd330/x64/pulumi up --non-interactive --yes --skip-preview --event-log /tmp/p-it-fv-az1444--user-c16efb32-events.json' in '/tmp/p-it-fv-az1444--user-c16efb32-540467627'
    command.go:98: Invoke '/opt/hostedtoolcache/pulumi/3.153.0-alpha.xf5bd330/x64/pulumi up --non-interactive --yes --skip-preview --event-log /tmp/p-it-fv-az1444--user-c16efb32-events.json' failed: exit status 255
Updating (p-it-fv-az1444--user-c16efb32):
    pulumi:pulumi:Stack okta-user-p-it-fv-az1444--user-c16efb32  warning: using pulumi-resource-okta from $PATH at /home/runner/work/pulumi-okta/pulumi-okta/bin/pulumi-resource-okta

 +  pulumi:pulumi:Stack okta-user-p-it-fv-az1444--user-c16efb32 creating (0s) warning: using pulumi-resource-okta from $PATH at /home/runner/work/pulumi-okta/pulumi-okta/bin/pulumi-resource-okta
 +  pulumi:pulumi:Stack okta-user-p-it-fv-az1444--user-c16efb32 creating (0s) warning: Could not find entry point '/tmp/p-it-fv-az1444--user-c16efb32-540467627/bin/index.js' specified in package.json; using '/tmp/p-it-fv-az1444--user-c16efb32-540467627' instead
@ updating....
 +  pulumi:pulumi:Stack okta-user-p-it-fv-az1444--user-c16efb32 creating (1s) (node:20713) [DEP0128] DeprecationWarning: Invalid 'main' field in '/tmp/p-it-fv-az1444--user-c16efb32-540467627/package.json' of 'bin/index.js'. Please either fix that or report it to the module author
 +  pulumi:pulumi:Stack okta-user-p-it-fv-az1444--user-c16efb32 creating (1s) (Use `node --trace-deprecation ...` to show where the warning was created)
 +  random:index:RandomPet random-first-name creating (0s) 
 +  random:index:RandomString random creating (0s) 
 +  random:index:RandomPet random-last-name creating (0s) 
 +  random:index:RandomPet random-first-name created (0.00s) 
 +  random:index:RandomPet random-last-name created (0.00s) 
 +  random:index:RandomString random created (0.00s) 
 +  pulumi:pulumi:Stack okta-user-p-it-fv-az1444--user-c16efb32 creating (1s) warning: using pulumi-resource-okta from $PATH at /home/runner/work/pulumi-okta/pulumi-okta/bin/pulumi-resource-okta
@ updating....
 +  okta:user:User example-user creating (0s) 
 +  okta:user:User example-user creating (0s) error:   sdk-v2/provider2.go:515: sdk.helper_schema: failed to create user: the API returned an error: Api validation failed: customProperty. Causes: errorSummary: customProperty: Property customProperty is required: provider=okta@4.15.0-alpha.1740550429
 +  okta:user:User example-user creating (0s) error: 1 error occurred:
 +  okta:user:User example-user **creating failed** error: 1 error occurred:
 +  pulumi:pulumi:Stack okta-user-p-it-fv-az1444--user-c16efb32 creating (1s) error: update failed
 +  pulumi:pulumi:Stack okta-user-p-it-fv-az1444--user-c16efb32 **creating failed** 1 error; 3 warnings; 2 messages
Diagnostics:
  okta:user:User (example-user):
    error:   sdk-v2/provider2.go:515: sdk.helper_schema: failed to create user: the API returned an error: Api validation failed: customProperty. Causes: errorSummary: customProperty: Property customProperty is required: provider=okta@4.15.0-alpha.1740550429
    error: 1 error occurred:
    	* failed to create user: the API returned an error: Api validation failed: customProperty. Causes: errorSummary: customProperty: Property customProperty is required

  pulumi:pulumi:Stack (okta-user-p-it-fv-az1444--user-c16efb32):
    warning: using pulumi-resource-okta from $PATH at /home/runner/work/pulumi-okta/pulumi-okta/bin/pulumi-resource-okta
    warning: Could not find entry point '/tmp/p-it-fv-az1444--user-c16efb32-540467627/bin/index.js' specified in package.json; using '/tmp/p-it-fv-az1444--user-c16efb32-540467627' instead
    warning: using pulumi-resource-okta from $PATH at /home/runner/work/pulumi-okta/pulumi-okta/bin/pulumi-resource-okta
    error: update failed

    (node:20713) [DEP0128] DeprecationWarning: Invalid 'main' field in '/tmp/p-it-fv-az1444--user-c16efb32-540467627/package.json' of 'bin/index.js'. Please either fix that or report it to the module author
    (Use `node --trace-deprecation ...` to show where the warning was created)

Resources:
    + 4 created

Duration: 3s


    command.go:118: Wrote output to /tmp/p-it-fv-az1444--user-c16efb32-540467627/command-output/pulumi-update-initial.20250226-062244.c8c1f.log
    program.go:1473: Destroying stack
    command.go:46: **** Invoke '/opt/hostedtoolcache/pulumi/3.153.0-alpha.xf5bd330/x64/pulumi destroy --non-interactive --yes --skip-preview' in '/tmp/p-it-fv-az1444--user-c16efb32-540467627'
    command.go:118: Wrote output to /tmp/p-it-fv-az1444--user-c16efb32-540467627/command-output/pulumi-destroy.20250226-062245.b4cb5.log
    command.go:46: **** Invoke '/opt/hostedtoolcache/pulumi/3.153.0-alpha.xf5bd330/x64/pulumi stack rm --yes' in '/tmp/p-it-fv-az1444--user-c16efb32-540467627'
    command.go:118: Wrote output to /tmp/p-it-fv-az1444--user-c16efb32-540467627/command-output/pulumi-stack-rm.20250226-062245.65ad8.log
    program.go:856: 
        	Error Trace:	/home/runner/go/pkg/mod/github.com/pulumi/pulumi/pkg/v3@v3.148.0/testing/integration/program.go:856
        	            				/home/runner/work/pulumi-okta/pulumi-okta/examples/examples_nodejs_test.go:31
        	Error:      	Received unexpected error:
        	            	running test preview, update, and edits: initial failure: [/opt/hostedtoolcache/pulumi/3.153.0-alpha.xf5bd330/x64/pulumi up --non-interactive --yes --skip-preview --event-log /tmp/p-it-fv-az1444--user-c16efb32-events.json] did not succeed after 1 tries
        	Test:       	TestAccUserTs
--- FAIL: TestAccUserTs (29.74s)
FAIL
```

--- a/provider/provider_program_test.go
+++ b/provider/provider_program_test.go
@@ -28,7 +28,11 @@ var programs = []string{
 	"test-programs/policy_rulepassword",
 	"test-programs/auth_server",
 	"test-programs/auth_serverscope",
-	"test-programs/index_userschemaproperty",
+
+	// This test races with TestAccUserTs test, in case the userschema property is defined,
+	// the TestAccUserTs test sporadically fails.
+	// "test-programs/index_userschemaproperty",
+
 	"test-programs/app_oauth",
 	"test-programs/user",
 }

